jira/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,52 @@
+= Jira Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Jira (Jira Connector) synchronizes data and automates business processes between Jira and third-party applications, either on-premises or in the cloud.
+
+Jira is a software development platform to help agile product development teams triage and track their work from inception to launch. Jira Connector enables organizations of all sizes to establish scalable and sustainable product development workflows by connecting Jira to other tools in the organization such as the customer service platforms, project management tools, and others.
+
+For compatibility information and fixed issues, see the Jira Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Jira APIs
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* Credentials to access the target Jira resource
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases for the Connector
+
+Automate business processes by integrating Jira with other IT tools and systems in your enterprise such as messaging, CRM, messaging, and others. 
```

Comment:
@valkyrie69 it says messaging twice. Can we change one of them to "customer service"?